### PR TITLE
Update technical.md

### PR DESCRIPTION
**Pull request by cebriggs7135**
@pecan88 @schwenk102 Please review and confirm accuracy, readability & links.

### CODE REVIEW

File: technical.md [hunk 1]
```diff
@@ -44,17 +44,19 @@
 <details>
 <summary>HuBMAP Data Validation</summary>
 
--   HuBMAP Data Validation is a continuously improving process that starts with data definitions including HuBMAP data QA/QC [standards for donor, sample, tissue](https://portal.hubmapconsortium.org/docs/metadata), and [assay](https://portal.hubmapconsortium.org/docs/assays) being created by a set of corresponding small teams under the Data Coordination Working Group.
+-   HuBMAP Data Validation is a continuously improving process that starts with defining QC/QA standards and establishing definitions for donor, sample and assay metadata. Standards, definitions, metadata schema and data directory schema are created by teams under the Data Coordination Working Group. The metadata schema may be accessed in Github via links from the [metadata page](https://portal.hubmapconsortium.org/docs/metadata) and downloaded as Excel spreadsheets. These spreadsheet templates have dropdown menus of constrained input for certain metadata fields.
```
Comment (mccalluc) [inline technical.md]:
```suggestion
-   HuBMAP Data Validation is a continuously improving process that starts with defining QC/QA standards and establishing definitions for donor, sample and assay metadata.
Standards, definitions, metadata schema and data directory schema are created by teams under the Data Coordination Working Group.
Metadata schemas are available [here](https://hubmapconsortium.github.io/ingest-validation-tools/),
along with Excel templates with dropdowns for data entry.
```
- Suggest adding returns here to keep the diff readable.
- Suggest getting rid of `/docs/metadata`.

File: technical.md [hunk 2]
```diff
@@ -44,17 +44,19 @@
...
+
     
--   Data providers format their data according to [these QA/QC standards](https://portal.hubmapconsortium.org/docs/assays) for submission, which are validated on ingest. Data providers also share their local QA/QC results as part of their data submission
+-   Data providers format their data and metadata files according to the metadata and data directory schema specifications for each assay type. Required formats for metadata field input are described in the Github page for each assay-specific metadata schema. Data providers also include the required QA/QC assessments of their data as components of the submission.
```
Comment (mccalluc) [inline technical.md]:
```suggestion
-   Data providers format their data and metadata files according to the metadata and data directory schema specifications for each assay type. Required formats for metadata field input are described in the [Github page for each assay-specific metadata schema](https://hubmapconsortium.github.io/ingest-validation-tools/). Data providers also include the required QA/QC assessments of their data as components of the submission.
```

File: technical.md [hunk 3]
```diff
@@ -44,17 +44,19 @@
...
     
 -   HuBMAP staff conduct 178 (and growing) automated and manual QA/QC checks as part of the data submission & publication process. Manual validation steps are being automated as development capacity allows.
     
--   Prior to publication, each dataset is reviewed in the Data Portal logged in as HuBMAP Members and is formally approved by the data providing institution and one or more HIVE members. Data providers must also QA/QC check spatial and semantic metadata using the [CCF EUI](https://portal.hubmapconsortium.org/ccf-eui).
+-   Prior to publication, each dataset is formally approved by the data-providing institution and one or more HIVE members. Data providers must also confirm the quality of spatial and semantic metadata using the [CCF EUI](https://portal.hubmapconsortium.org/ccf-eui).
```
Comment (mccalluc) [inline technical.md:62]: Unfortunately, what precisely the approval process is still unclear.

File: technical.md [hunk 4]
```diff
@@ -115,3 +117,9 @@
 
 -   We also automatically collect and display [HuBMAP-generated and referenced publications](https://scholar.google.com/citations?user=CtGSN80AAAAJ&hl=en) using Google Scholar.
 </details>
+
+
+<details>
+<summary>HuBMAP Assays and Metadata</summary>
+    
+-   HuBMAP assay documentation and links to assay metadata template are accessible from the [metadata page](https://portal.hubmapconsortium.org/docs/metadata).
```
Comment (mccalluc) [inline technical.md]:
```suggestion
```
- Please balance tags.
- Again, suggest deleting `docs/metadata`

**Comment by mccalluc** [comment]
Thanks for taking the time to tidy this up!

File: technical.md [hunk 5]
```diff
@@ -19,7 +19,7 @@
 <details>
 <summary>HuBMAP Data Ingest</summary>
     
--   HuBMAP HIVE is responsible to produce and manage data ingest processes and associated software in collaboration with the Data Providers, whereas HuBMAP Data Providers are responsible to produce data and metadata in collaboration with the HIVE. These have been very productive activities and are rapidly evolving to scalable data ingest processes with the current state outlined below.
+-   HuBMAP HIVE is responsible for producing and managing data ingest processes and associated software in collaboration with the Data Providers. HuBMAP Data Providers are responsible for producing data and metadata in collaboration with the HIVE. These processes are rapidly evolving into scalable ingest processes.
```
Comment (cebriggs7135) [inline technical.md]:
```suggestion
-   HuBMAP HIVE is responsible for producing and managing data ingest processes and associated software in collaboration with the Data Providers. HuBMAP Data Providers are responsible for producing data and metadata in collaboration with the HIVE. These processes are rapidly evolving into scalable pipelines.
```

File: technical.md (hunk 3 above)
Comment (cebriggs7135) [reply technical.md:62]: @mccalluc Peter says he and Melissa wrote the current text and that I have "broad authority" over this part of the documentation. I'll check with Peter to find out who they had review their text. I value your opinion so if you have any comments or edits, please add them.

Comment (cebriggs7135) [reply technical.md:62]: @mccalluc Excellent fixes! Thanks.

**Review by pecan88 — APPROVED** [review]

**Pull request merged**